polysemy-zoo.cabal

```diff
@@ -99,7 +99,7 @@ test-suite polysemy-zoo-test
     , hedis >=0.10 && <0.13
     , hspec >=2.6.0 && <3
     , mtl >=2.0.1.0 && <3.0.0.0
-    , polysemy >=0.5.1.0
+    , polysemy >=0.7.0.0
```

Comment:
Nice catch! I don't know how I missed this, but it probably breaks stack. Whoops!